keyboards/edc40/info.json

```
@@ -0,0 +1,54 @@
+{
+    "keyboard_name": "edc40",
+    "url": "",
+    "maintainer": "OJ",
```

This is meant to reference your GitHub username.

```suggestion
    "maintainer": "ojthetiny",
```